client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/notices/notices.tsx

```diff
@@ -11,6 +18,17 @@ const Notices = ( {
 		noticeOperations.removeNotice( id );
 	};
 
+	useEffect( () => {
+		const lastNotice = noticeList.at( -1 ) as Notice;
```

Comment:
It might be better to use `noticeList[ noticeList.length - 1 ]` or `noticeList.slice( -1 )` to get the last one since `.at()` since it only supported after safari 15.4 and above. I used BrowserStack to test it and looks like we won't add the polyfill of this feature to calypso

<img width="461" alt="image" src="https://user-images.githubusercontent.com/13596067/227838559-45430110-f549-42ba-a50e-9bfe0bedd410.png">

Reply:
Thanks!